plugins/MauticCrmBundle/Form/Type/IntegrationCampaignsTaskType.php

```diff
@@ -89,15 +89,15 @@ function ($validateMe, ExecutionContextInterface $context) {
     /**
      * {@inheritdoc}
      */
-    public function setDefaultOptions(OptionsResolverInterface $resolver)
```

Comment:
Use `array_flip()` on the member choices on line 71 and ensure that all choice type choices in this bundle have the values and labels flipped.

Reply:
Fixed in eb4f00a1e0a52ca435769071b7a18dda1d225b1f

Reply:
I forgot to tell you that you should also add `choices_as_values` option as described in https://docs.google.com/document/d/1lHNK8hHQtvyWlz9OtfAoLupDVb5-QunQjq-HAGlAsS4/edit#

I thought I added it to all comments on this topic I did yesterday but I must have forgotten it here.

Reply:
No worries. Fixed in bfc595560a387c4015f4f14fd3992c57679f83cf